### ISSUE TITLE
noUnusedVariable too aggressive?

### ISSUE THREAD

**Issue by NickHeiner** [palantir/tslint#478]
**tslint version**
```
λ tslint --version
2.3.1-beta
```

**tslint.json**
```json
{
  "rules": {
    "no-unused-variable": true
  }
}
```

**index.ts**
```ts
'use strict';

let x = 0;

console.log(x);
```

**Result**
```
λ tslint --file index.ts --config tslint.json 
index.ts[3, 5]: unused variable: 'x'
```

This surprises me. Is this intended behavior? Or am I missing something? I would consider the variable `x` to be used because I am passing it to a function. 

If I add an `exports` to the file, then the problem disappears.

**Comment by leeavital** [comment]
This might be related to https://github.com/palantir/tslint/issues/325.

**Comment by adidahiya** [comment]
@JKillian can you please verify if this has been fixed?

**Comment by JKillian** [comment]
:+1: Not getting the unused variable warning anymore
